nym-vpn-core/crates/nym-vpnd/src/main.rs

```diff
@@ -122,7 +122,13 @@ async fn run_inner_async(
     network_env: Network,
     logging_setup: Option<LoggingSetup>,
 ) -> anyhow::Result<Option<WorkerGuard>> {
-    network_env.check_consistency().await?;
+    match network_env.check_consistency().await {
+        // This is probably because of network unavailability, so consistency can't be double checked,
+        // but we shouldn't fail just because of that
+        Err(e) => tracing::warn!("Could not check consistency: {e:?}"),
```

Comment:
Can we please unroll this:

```rs
use nym_common::ErrorExt;

tracing::warn!("Could not check consistency: {}", e.display_chain())
```

Reply:
I've unrolled using `anyhow`'s internal unroll, as our `ErrorExt` is not currently compatible with `anyhow`'s `Error`.